workflows/templates/unresolved-package-handler-template.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: unresolved-package-handler
+  annotations:
+    thoth-station.ninja/template-version: 0.1.0
+  labels:
+    app: thoth
+    component: unresolved-package-handler
+
+spec:
+  templates:
+    - name: unresolved-package-handler
+      inputs:
+        parameters:
+          - name: THOTH_DOCUMENT_ID
+          - name: "IMAGE_STREAM_REGISTRY"
```

Comment:
we can get rid of `IMAGE_....` as the imagestream will live in the same namespace as the workflow instance, so it will be found without giving any registry information.

Reply:
is it something we do already? because meanwhile it has to work with adviser: https://github.com/thoth-station/adviser/pull/983